### ISSUE TITLE
Train VC model from scratch for 44.1 KHz output from DiT

### ISSUE THREAD

**Issue by GiriR45** [Plachtaa/seed-vc#158]
Hello, 

Thanks for open-sourcing the work. I've been training and fine-tuning this model and was able to get very good results at 22 KHz. 

I running an experiment on training the model in 44KHz to reproduce. I've followed the issues https://github.com/Plachtaa/seed-vc/issues/134 and https://github.com/Plachtaa/seed-vc/issues/133 to make changes to the config. 

And I've started training from scratch on ~1000 hours of data for Indian languages. So far I've trained for 310k steps, The audio quality is significantly improved, compared to the inference samples of the initial checkpoints but still, there is minimal graining noise/ improper or incomplete vocal reconstruction, will training further improve the synthesis quality or is there any mistakes that I've made in the config? 

I'm attaching a couple of synthesis samples of the initial steps and a recent one. I'm also attaching the config file which I used for training. 

Can you please go through it once and provide your feedback? Thanks!

Changes made: 
Speech Encoder: Whisper-small to Whisper-large-v3-turbo, changed the in_length of the length_regulator to 1280.
Vocoder: BigVGAN_v2_22KHz to BigVGAN_v2_44KHz.
Modified the preprocess params for 44.1KHz based on the values on SVC config.

[seedvc_44khz.zip](https://github.com/user-attachments/files/19752852/seedvc_44khz.zip)

**Comment by Plachtaa** [comment]
Model performance mainly mainly depends on your experiment setup up and dataset, which I cannot help with

**Comment by GUUser91** [comment]
If GiriR45 is training a V1 model, he should check the ft_dataset.py file from the data directory, there's several lines like
```
class FT_Dataset(torch.utils.data.Dataset):
        sr=22050,
```
and

```
if __name__ == "__main__":
    data_path = "./example/reference"
    sr = 22050
    spect_params = {
        "n_fft": 1024,
        "win_length": 1024,
        "hop_length": 256,
        "n_mels": 80,
        "fmin": 0,
        "fmax": 8000,
    }
```
I think the sr info should be 44100 and the spect_params info should match bigvgan_v2_44khz_128band_512x or whatever GiriR45 is using.

Also he should resume a checkpoint model that has loss below 1 and then increase batch_length: and max_len: for better intelligible speech results.
